main.go

```diff
@@ -194,9 +194,11 @@ func main() {
 
 	setupHTTP(ctx, srv.Cert(), srv.ModTime(), errCh)
 
-	if err := srv.ingest(ctx, api, facility); err != nil {
-		logger.Error(err)
-		panic(err)
+	if env.Bool("CACHER_NO_INGEST") {
```

Comment:
Should there be a `not` of some kind here?

Reply:
:facepalm: indeed

Reply:
oh now I remember why I didn't want to do this originally. The `if` branch should be exceptional/abnormal cases but that would not be the case here.

Reply:
@nshalman went back to doing the skip in ingest.go not main.go for ^ and more reasons (in the commit message).

Reply:
👍